package.json

```diff
@@ -8,7 +8,10 @@
     "test": "jest"
   },
   "dependencies": {
+    "@react-native-community/masked-view": "0.1.10",
```

Comment:
This came from running 

`expo install react-native-gesture-handler react-native-reanimated react-native-screens react-native-safe-area-context @react-native-community/masked-view`

which came from the React Navigation docs:
https://reactnavigation.org/docs/getting-started#installing-dependencies-into-an-expo-managed-project